### PR TITLE
Documentation - Add Info about Plotting Board

### PR DESCRIPTION
**Pull request by PulsarNeutronStar**
**When merged this pull request will:**
Adds missing Information to MapTools/PlottingBoard

### CODE REVIEW

File: docs/wiki/feature/maptools.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ If you are equipped with a vanilla GPS it will be shown on the map. (You don't n
 - Press the self interaction key <kbd>Ctrl</kbd> + <kbd>⊞&nbsp;Win</kbd> (ACE3 default key bind `Self Interaction Key`).
 - Select `Show Plotting Board`.
 - Note that you can drag the Plotting Board around with <kbd>LMB</kbd> and rotate the different parts of the Plotting Board, each independently from each other, with <kbd>Ctrl</kbd> + <kbd>LMB</kbd>.
+- When a new marker (Local or Groupchannel) on the inner circle of the Plotting board is created, the marker will move accordingly when the Plotting Board gets moved or rotated.
```
Comment (Drofseh) [inline docs/wiki/feature/maptools.md:52]: Groupchannel

Comment (PulsarNeutronStar) [reply docs/wiki/feature/maptools.md:52]: I dont follow?

Comment (Drofseh) [reply docs/wiki/feature/maptools.md]: There should be a space between Group and channel

Comment (PulsarNeutronStar) [reply docs/wiki/feature/maptools.md]: done

File: docs/wiki/feature/maptools.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ If you are equipped with a vanilla GPS it will be shown on the map. (You don't n
 - Press the self interaction key <kbd>Ctrl</kbd> + <kbd>⊞&nbsp;Win</kbd> (ACE3 default key bind `Self Interaction Key`).
 - Select `Show Plotting Board`.
 - Note that you can drag the Plotting Board around with <kbd>LMB</kbd> and rotate the different parts of the Plotting Board, each independently from each other, with <kbd>Ctrl</kbd> + <kbd>LMB</kbd>.
+- When a new marker (Local or Group Channel) on the inner circle of the Plotting board is created, the marker will move accordingly when the Plotting Board gets moved or rotated.
```
Comment (johnb432) [inline docs/wiki/feature/maptools.md:52]: The structure of the sentence feels off:
```suggestion
- When a new marker (Local or Group Channel) is created on the inner circle of the Plotting board, the marker will move accordingly when the Plotting Board gets moved or rotated.
```

**Comment by LinkIsGrim** [comment]
Is it too much to ask to get this in the field manual as well?

**Comment by PulsarNeutronStar** [comment]
> Is it too much to ask to get this in the field manual as well?

i can look into it, how / where though?

**Comment by LinkIsGrim** [comment]
https://github.com/acemod/ACE3/tree/master/addons/fieldmanual component, https://github.com/acemod/ACE3/pull/9482 for example

**Review by PabstMirror — APPROVED** [review]

**Comment by PabstMirror** [comment]
We can handle a field manual in another pr

**Review by LinkIsGrim — APPROVED** [review]

**Comment by PulsarNeutronStar** [comment]
> We can handle a field manual in another pr

yea sorry, i have to many other projects open right now but im happy to help out with the field manual once i can

**Pull request merged**